src/Illuminate/View/ComponentAttributeBag.php

```diff
@@ -38,13 +39,13 @@ public function __construct(array $attributes = [])
     }
 
     /**
-     * Get all of the attribute values.
+     * Get all the attribute values.
      *
      * @return array
      */
     public function all()
     {
-        return $this->attributes;
+        return $this->toArray();
```

Comment:
As the justification is to assimilate/normalize/standardize this class with its peers on the framework, almost all methods named `all()` do not simply defer to a single method call without some sort of logic (the exception being `Illuminate\Queue\Failed\FileFailedJobProvider`)

And the two classes containing both a `all()` method and a `toArray()` method, the `toArray()` method calls the `all()` one instead of the proposed implementation. These classes being: `Illuminate\Http\Request` and `Illuminate\Support\ValidatedInput`.

So I'd suggest keeping `all()` implementation's as before, and just calling `$this->all()` from`toArray()`.